distribution/osbuild-composer.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=OSBuild Composer
+After=multi-user.target
+# Running osbuild-composer without the worker is useless
+# therefore it is a hard requirement
+Requires=osbuild-worker.service osbuild-composer.socket
+
+[Service]
+Type=simple
+ExecStart=/usr/lib/osbuild-composer/osbuild-composer
+# /run/..
+RuntimeDirectory=osbuild-composer weldr
+# /var/lib/..
+StateDirectory=osbuild-composer
+WorkingDirectory=/usr/lib/osbuild-composer/
+User=_osbuild-composer
+Restart=on-failure
+
+# Worker will contact composer immediately after start-up
```

Comment:
Hm, I'd drop the comment. We don't know about the worker (when they come online), but most probably we do want to start this eagerly so it can download its metadata etc before someone tries to use it from cockpit (so I agree with the install section).